README.md

```diff
@@ -66,17 +66,32 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
-* Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
+* Create a kubeconfig file which has the context of all the clusters (including the Broker cluster). If KUBECONFIG environment variable set with the path to all clusters (like KUBECONFIG=/home/user/.config/east/auth/kubeconfig-dev:/home/user/.config/west/auth/kubeconfig-dev), you could use the below command.
 ```
-subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME> --clusterid  <CLUSTER-ID-FOR-TUNNELS>
+kubectl config view --flatten > merged_kubeconfig
```

Comment:
But this does not work for openshift configs, all clusters will have context "admin".

We need a solution for that as part of #210